vioscsi/vioscsi.c

```diff
@@ -330,7 +330,7 @@ BOOLEAN VioScsiReadRegistry(
            (PVOID)pBuf,
            sizeof(ULONG));
     adaptExt->max_physical_breaks = min(
-                                        max(SCSI_MINIMUM_PHYSICAL_BREAKS, adaptExt->max_physical_breaks),
+                                        max(SCSI_MINIMUM_PHYSICAL_BREAKS, adaptExt->max_physical_breaks + 1),
```

Comment:
While I have this in the commit, there's no reason we should do so other than the fact that we set number of pages / physical breaks in registry (so it takes less work to calculate here than in the max_sectors case).